test/Complex.lean

```diff
@@ -0,0 +1,20 @@
+import Mathlib.Data.Complex.Basic
+import Mathlib.Algebra.DualNumber
+
+open DualNumber
+
+private axiom test_sorry : ∀ {α}, α
+unsafe def testRepr (c : ℂ) (s : String) : Lean.Elab.Command.CommandElabM Unit :=
+unless toString (repr c) = s do throwError "got {repr c}"
+
+unsafe def testDualNumberRepr (c : ℂ[ε]) (s : String) : Lean.Elab.Command.CommandElabM Unit :=
+unless toString (repr c) = s do throwError "got {repr c}"
+
+-- Test base complex number representation
+run_cmd unsafe testRepr 0 "Real.ofCauchy (sorry /- 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ... -/) + Real.ofCauchy (sorry /- 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ... -/)*I"
```

Comment:
Usually we'd just test these things with
```lean
/-- a code action will replace this docstring -/
#guard_msgs in
#check (0 : ℂ)
```

Reply:
I couldn't quite get this working (I may have missed which code action to use, but the build failed on GitHub) so I applied this to what I had. Let me know if this is sufficient.